deploy/helm/flightctl/templates/flightctl-alert-exporter-config.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.alertexporter.enabled }}
```

Comment:
Naming is a bit inconsistent, since "alertmanager" is a single word, but "alert exporter" is not.

With that assumption, `alertexporter` should be `alertExporter` as that's the convention for the other settings, etg `cliArtifacts`

Alternatively, we could maybe consider naming "alert exporter" as "alertexporter".

Reply:
Good call, will fix